### PR TITLE
Fix config file modification

### PR DESCRIPTION
**Pull request by GrygrFlzr**
- Fixes missing `tailwindcss` invocation on Windows
- Make sure to use `tailwind.config.cjs`

### CODE REVIEW

File: preset.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ const globalCSS = `@tailwind base;
 const addTailwind = (otherPlugins) => `plugins: [
 		// Some plugins, like postcss-nested, need to run before Tailwind
 		
-		tailwindcss,
+		tailwindcss("./tailwind.config.cjs"),
```
Comment (babichjacob) [inline preset.ts]: Did you see a situation where this was needed? We undid this per #9.

Comment (GrygrFlzr) [reply preset.ts]: Ah, I missed that. No, that was under the assumption that it still needed it, will re-test without.

Comment (GrygrFlzr) [reply preset.ts]: Alright, reverted and dark mode definitely works without the explicit config file.

**Comment by babichjacob** [comment]
Thank you! 😃

**Pull request merged**